components/freertos/include/freertos/projdefs.h

```diff
@@ -42,7 +42,7 @@ definition here is not suitable for your application. */
 #endif
 
 #ifndef pdTICKS_TO_MS
-	#define pdTICKS_TO_MS( xTicks )   ( ( uint32_t ) ( xTicks ) * 1000 / configTICK_RATE_HZ )
+	#define pdTICKS_TO_MS( xTicks )   ( ( TickType_t ) ( ( uint64_t ) ( xTicks ) * 1000 / configTICK_RATE_HZ ) )
```

Comment:
We updated the macro in commits ff56848a0eca09865ec92f1c72d55e39b3984392 and 3f2f6d0cf5438037f891ce9e483af1709ad8a33d to match upstream FreeRTOS v10.4.3:

```c
/* Converts a time in milliseconds to a time in ticks.  This macro can be
 * overridden by a macro of the same name defined in FreeRTOSConfig.h in case the
 * definition here is not suitable for your application. */
#ifndef pdMS_TO_TICKS
    #define pdMS_TO_TICKS( xTimeInMs )    ( ( TickType_t ) ( ( ( TickType_t ) ( xTimeInMs ) * ( TickType_t ) configTICK_RATE_HZ ) / ( TickType_t ) 1000U ) )
#endif
```
As per upstream, I think it would be better to override this macro in [`FreeRTOSConfig.h`](https://github.com/espressif/esp-idf/blob/master/components/freertos/esp_additions/include/freertos/FreeRTOSConfig.h) so that we can keep `projdefs.h` clean

Reply:
Sure, work on copy at FreeRTOSConfig.h is really better way and its workarund for this problem. Any way it still bug on code pdTICKS_TO_MS on projdefs.h and this is poor solution.

Reply:
@lisekt84 sorry, my mistake. I mixed up `pdMS_TO_TICKS` and `pdTICKS_TO_MS`. `pdTICKS_TO_MS` doesn't exist in upstream FreeRTOS so it should be OK to update it directly in `projdefs.h`. I'll merge your commit internally asap.

Reply:
I also checked pdMS_TO_TICKS and this define is wrong too.
I think the perfect way is set the comment about this problem.
Problem start at 71 minutes and 35 sec from start (UINT32_MAX / 1000).